pkg/analyze/analyze.go

```diff
@@ -288,10 +299,45 @@ func (a *Analyzer) PrintTopValues(displayRecord map[netip.Prefix]time.Time, sort
 				connection = "     "
 			}
 		}
-		a.logger.Printf("%s%16s%s: %7s %3d %7s %s (from %s, last accessed %s)%s\n", fmtStart, key, connection, humanize.IBytes(total), reqTotal,
+		a.logger.Printf("%s%16s%s: %7s %3d %7s %s (from %s, last accessed %s)%s\n", fmtStart, key.prefix, connection, humanize.IBytes(total), reqTotal,
 			humanize.IBytes(average), last, lastUpdateTime, lastAccessTime, fmtEnd)
 		if displayRecord != nil {
-			displayRecord[key] = ipStats.LastURLAccess
+			displayRecord[key.prefix] = ipStats.LastURLAccess
 		}
 	}
 }
+
+func (a *Analyzer) GetCurrentServers() []string {
+	if a.Config.UseLock() {
+		a.mu.Lock()
+		defer a.mu.Unlock()
+	}
+	servers := make([]string, 0, len(a.serverIpInfo))
+	for sp := range a.serverIpInfo {
+		servers = append(servers, sp.server)
+	}
+	return servers
```

Comment:
Sure we don't run deduplication on this returned list?

Reply:
Oops, forgot about this when changing map key type.